spec/lib/adv_ctc_irs1040_pdf_spec.rb

```diff
@@ -0,0 +1,255 @@
+require "rails_helper"
+
+require "rails_helper"
+
+RSpec.describe AdvCtcIrs1040Pdf do
+  include PdfSpecHelper
+
+  let(:pdf) { described_class.new(submission) }
+  let(:submission) { create :efile_submission, :ctc }
+
+  describe "#output_file" do
+    context "with an empty submission record" do
+
+      # "clear out" submission so we can see the empty state of the PDF
+      before do
+        submission.intake.update(email_address: nil, phone_number: nil, sms_phone_number: nil, primary_first_name: "", primary_last_name: "", primary_ssn: "", claim_owed_stimulus_money: "no" )
+        submission.address.destroy!
+        submission.intake.bank_account.destroy!
+        submission.intake.dependents.destroy_all
+        submission.tax_return.update(filing_status: nil)
+        submission.reload
+      end
+
+      it "returns a pdf with default fields and values" do
+        output_file = pdf.output_file
+        result = non_preparer_fields(output_file.path)
```

Comment:
Why are we removing the intake_specialist fields? Would just using `filled_in_values` in PdfSpecHelper accomplish the same thing in this test?